lib/v2/finology/tag.js

```diff
@@ -0,0 +1,33 @@
+const { error } = require('winston');
```

Comment:
Please use the exported logger instead of creating a new instance.
https://github.com/DIYgod/RSSHub/pull/12993#discussion_r1291316342 https://github.com/DIYgod/RSSHub/pull/12941#discussion_r1285912877